src/test/java/com/ericsson/ei/flowtests/FlowTestBase.java

```diff
@@ -209,14 +209,14 @@ private long countProcessedEvents(String database, String collection) {
         return table.count();
     }
 
-    private void waitForEventsToBeProcessed(int eventsCount) {
+    protected void waitForEventsToBeProcessed(int eventsCount) {
         // wait for all events to be processed
         long processedEvents = 0;
         while (processedEvents < eventsCount) {
             processedEvents = countProcessedEvents(database, event_map);
             LOGGER.info("Have gotten: " + processedEvents + " out of: " + eventsCount);
             try {
-                TimeUnit.MILLISECONDS.sleep(3000);
+                TimeUnit.MILLISECONDS.sleep(10000);
```

Comment:
Maybe this waitTime should be defined in properties file.

Reply:
Is that really necessary  with such a specific timer in a test? If there were multiple tests using the same timer then i guess it could be put out but at the moment i dont see the win of it.